### DISCUSSION TITLE
咸鱼肆分肆基地

### DISCUSSION THREAD

**Discussion by smd1121** [SaltyfishShop/big_discusson#2] [Information]
咸鱼肆除了主肆 **(QQ 群: 282781045)** 之外，大家也搞出了专门性的分肆 QWQ

分肆和咸鱼肆并无必然联系，分肆完全由创建者拥有和管理。分肆也不一定需要是 QQ 群。主肆中的肆友都可以建立分肆QWQ（但请不要做会损害其他肆友利益的事喔）！

如果分肆的创建者愿意公开分肆，可以考虑把分肆放在这里，以便大家了解和加入！

**注意：主肆禁止讨论政治敏感话题。如果确有需求，请移步 [咸鱼肆激情冲浪分肆](https://github.com/SaltyfishShop/big_discusson/discussions/2#discussioncomment-4247105)。谢谢！**

**Comment by smd1121** [comment]
### 咸鱼肆激情冲浪分肆
QQ 群：777322861

该分肆用于讨论任何政治敏感话题；同时主肆禁止讨论政治敏感话题。

该分肆是为了保护咸鱼肆，以及保护那些并不想接触政治敏感话题的肆友而建立的。

**Comment by smd1121** [comment]
### 咸鱼肆 C++ 分肆

QQ 群：598167763

**该分肆禁水。**

**Comment by ruoxining** [comment]
### **咸鱼肆计网分肆**
群号：563160469
注：
这群怎么忽然变成了CS年级群啊啊啊啊啊
有很多人在里面提问实验
不建议分享不适合向校外公开传播的资料，感谢！🙏

**Comment by Zhangyc0** [comment]
咸鱼肆 Telegram 分肆
https://t.me/+_NIU_seeZVZhM2E1

**Reply by Zhangyc0** [reply]
进来记得介绍一下自己，说个id就行

**Comment by Guahao31** [comment]
咸鱼肆 **数逻祭祖分肆**：
QQ群 668015216
欢迎来对浙大计院硬件相关课程（尤其实验）激情建(kai)议(pen)～

**Comment by cxzhou35** [comment]
咸鱼肆 Vim 分肆：
QQ 群：700934087 (Vim 大法好！
不定期会分享一些 Vim 的实用技巧和花哨配置<del>(瞎折腾)</del>，欢迎感兴趣的同学来玩！

**Comment by tmplog** [comment]
咸鱼肆 Leetcode 分肆：
主页：https://github.com/SaltyfishShop/leetcode_subshop
QQ群：947232069
大家一起打周赛双周赛，上上分，不残酷
